pandas/io/json/_json.py

```diff
@@ -816,6 +827,8 @@ def close(self):
                 self.open_stream.close()
             except (IOError, AttributeError):
                 pass
+        for file_handle in self.file_handles:
+            file_handle.close()
```

Comment:
probably unrelated to the recent CI issues, but we should definitely close those handles.

Reply:
hmm, is there a ResoucceWarning?

Reply:
I haven't seen any when reading/writing json files